src/capi/azext_capi/custom.py

```diff
@@ -136,21 +107,61 @@ def _install_capz_components(cmd):
             output = subprocess.check_output(command, universal_newlines=True, stderr=stderr)
             logger.info("%s returned:\n%s", " ".join(command), output)
         except subprocess.CalledProcessError as err:
-            raise UnclassifiedUserFault("Can't locate a Kubernetes cluster") from err
+            raise UnclassifiedUserFault("Couldn't install capz components in current cluster") from err
 
 
 def create_management_cluster(cmd, yes=False):
     check_prereqs(cmd)
     msg = 'Do you want to initialize Cluster API on the current cluster?'
```

Comment:
If I don't have a Kubernetes cluster in the default context, I get this:

```
 % az capi management create
Command group 'capi management' is in preview and under development. Reference and support levels: https://aka.ms/CLI_refstatus
Do you want to initialize Cluster API on the current cluster? (y/N): y
Can't locate a kubernetes cluster
%
```

Maybe it would be more user-friendly to run the command `kubectl config current-context` and insert that name into the prompt, or just skip to create if there is no current context?

```
 % az capi management create
Do you want to initialize Cluster API on the current cluster "kind-capz"? (y/N): y
```

Reply:
I agree, what do you think of this flow:

User runs `az capi management create`
Backend:
We start by run `kubectl config current-context` to verify an existing context. After use context name to get cluster's name by `kubectl config get-contexts <current-context-name>` (The use of extra command is in case the context name is different from cluster name). If this yields positive outcome: 
```
> Do you want to initialize Cluster API on the current cluster "kind-capz"? (y/N):
```
else:
```
if: no current-context found.
> No Kubectl current-context found

if: current-context found but not cluster name within context:
> No cluster found within <current-context> context.

Last Line for both scenarios:
> Proceeding to create a new management cluster
```

Reply:
That sounds good. You're right that context != cluster, so your approach should cover all the bases.